SourceX/dx.cpp

```diff
@@ -261,7 +261,31 @@ void RenderPresent()
 #ifndef USE_SDL1
 	if (renderer) {
 		if (SDL_UpdateTexture(texture, NULL, surface->pixels, surface->pitch) <= -1) { //pitch is 2560
-			ErrSdl();
+			// We should supposedly handle SDL_RENDER_DEVICE_RESET, but I tried and it doesn't arrive
+			if (!strcmp(SDL_GetError(), "LockRect(): INVALIDCALL")) {

```

Comment:
I wonder if this could be detected similar to what was done with GetOutputSurface based on the state of the texture. could you check the state of SDL_WasInit() and see what returns?

Reply:
`0xe231` which is the same value as passed to `SDL_Init`

I guess that only the texture(s) get invalidated by the sleep, not the whole of SDL.